bootstrap/cmd/bootstrap/main.go

```diff
@@ -43,6 +44,14 @@ func main() {
 	}
 
 	if err := app.Run(s); err != nil {
-		log.Fatalf("%v\n", err)
+		if s.InCluster && s.KeepAlive {
+			log.Infof("Bootstrapper failed with error: %v\n", err)
+			log.Infof("Keeping pod alive...")
```

Comment:
Please log a description of why we're doing this so that it's easier for the user to understand what's happening